src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -291,6 +286,15 @@
              app-keyboard-listener (.addEventListener rn/app-state "change" dismiss-keyboard-fn)]
          #(.remove app-keyboard-listener))))
     (hot-reload/use-safe-unmount on-navigate-back)
+    (rn/use-effect
+     (fn []
+       (set-input-state
+        #(-> %
+             (controlled-input/set-upper-limit
+              (utils/cut-crypto-decimals-to-fit-usd-cents
+               (or default-limit-crypto token-balance)
+               conversion-rate)))))
```

Comment:
Just wondering if `conversion-rate` always uses the updated value and never refers to an old value, as it depends on token

Reply:
Yes, it will use the updated value as the token is changed 👍

Reply:
@smohamedjavid the comment from @mohsen-ghafouri is interesting, should we add `token` as dependecy for this `use-effect`? or directly `conversion-rate` and `token-balance`?

I'm just having some questions, not suggesting a change

Reply:
Thanks @smohamedjavid